drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -751,6 +756,7 @@ static int nrf5_init(const struct device *dev)
 
 	nrf5_radio->rx_on_when_idle = true;
 	nrf5_radio_cfg->irq_config_func(dev);
+	nrf5_data.channel = nrf_802154_channel_get();
```

Comment:
Actually, `nrf5_radio` points to `nrf5_data`. Please align the access for consistency:
```suggestion
	nrf5_radio->channel = nrf_802154_channel_get();
```

Reply:
updated, please take a look if it's correct now :)